### ISSUE TITLE
Creation of empty Migration

### ISSUE THREAD

**Issue by janpio** [prisma/prisma#6411]
## Problem

Prisma Migrate will not allow me to create an empty migration that I can fill with SQL myself. When I want to only execute SQL for information that is not stored in my Prisma Schema, that is a hassle.

## Suggested solution

Let me generate empty migrations.

## Alternatives

- Add a fake model to the schema, create the migration with `--create-only`, remove the fake model again, adapt the `migration.sql` file, run `npx prisma migrate dev`

**Comment by tomhoule** [comment]
`--create-only` works even with an empty schema. Is this issue about giving you a way to generate an empty migration without touching the dev database?

**Comment by janpio** [comment]
Yes, equivalent of creating the folder and empty file. Feel free to fix my misleading description above.

**Comment by tomhoule** [comment]
I think we can close this issue as a duplicate of https://github.com/prisma/prisma/issues/6411 then

**Comment by janpio** [comment]
That would be weird, as it's the same issue :D But if there is another one, sure.

**Comment by tomhoule** [comment]
Ugh, why does this happen to me all the time. I'll try to find the other one again :p

**Comment by TanguyP** [comment]
I have the opposite problem: if I've changed nothing in my `prisma.schema` file and run `prisma migrate dev --create-only`, Prisma will create an empty migration - which I don't want.

So maybe the problem originally described in this ticket is gone? Additionally (I think the following discussion makes sense as part of this ticket too), I would argue for an `--allow-empty` flag: if it's missing, Prisma would just tell you something like: ```Nothing to migrate. If you want to create an empty migration, add the `--allow-empty` flag```
